WordPress/UITests/Tests/DashboardTests.swift

```diff
@@ -28,4 +28,15 @@ class DashboardTests: XCTestCase {
             .goToPlanSelection()
             .verifyPlanSelectionScreenLoaded()
     }
+
+    func testPagesCardHeaderNavigation() throws {
```

Comment:
is this test also only for Jetpack? i see that the first test in this test class has a comment on it `// This test is JP only.` what do you think about adding a comment for the whole test class instead of per test so it's clearer that it's Jetpack only?

Reply:
Good point! Done in 865dd4194b6996bd167ef7cceef3c94d8c6ac269.